Source/FlavoredNeutrinoContainerInit.cpp

```diff
@@ -69,6 +112,16 @@ namespace
     *Usymmetric = 2. * (amrex::Random()-0.5);
   }
 
+// angular structure as determined by the Minerbo closure
+// Z is a parameter determined by the flux factor
+// mu is the cosine of the angle relative to the flux direction
+// Coefficients set such that the expectation value is 1
+  AMREX_GPU_HOST_DEVICE void minerbo_closure(Real* result, const Real Z, const Real mu){
+	Real minfluxfac = 1e-3;
+	*result = std::exp(Z*mu);
+	if(Z/3.0 > minfluxfac)
```

Comment:
the only bit I'm still uncertain about is where this 3 came from here in line 122 and in line 64 above?

Reply:
This just comes from the limit at small flux factor, where Z is approximately 3*fluxfac